opennms-doc/guide-admin/src/asciidoc/text/telemetryd/introduction.adoc

```diff
@@ -2,68 +2,80 @@
+_Listeners_ and attached _Parsers_ on one side, receive telemetry data transported over a specific protocol and parse the protocol according to its specification.
 On the other side, _Adapters_ pick up the parsed data and enrich it before persisting it.
 
-_Queues_ are used to transport the parsed telemetry data from _Parsers_ to _Adapters_ by binding a specific _Parser_ to a specific _Adapter_.
+_Queues_ transport the parsed telemetry data from _Parsers_ to _Adapters_ by binding a specific _Parser_ to a specific _Adapter_.
 
 === _Listeners_ and _Parsers_
-_Listeners_ are responsible for receiving sensor data from some external source and hand them off to _Parsers_ which handles the transport protocol format and extracts the transported data.
+_Listeners_ receive sensor data from an external source and hand them off to _Parsers_, which handle the transport protocol format and extract the transported data.
 
 A _Listener_ does the transport protocol handling like opening a socket and accepting incoming connections.
-The received messages are then passed to the parser configured for the _Listener_ which parses the concrete protocol implementation and extract all available information out of the received message.
+The received messages are then passed to the parser configured for the _Listener_ which parses the concrete protocol implementation and extracts all available information out of the received message.
 For further processing, the extracted data is handed over to the configured queue.
 
-For example: A _Listener_ may open a UDP socket for receiving packets and a assigned _Parser_ parses the incoming IPFix packets.
+For example: a _Listener_ may open a UDP socket for receiving packets and an assigned _Parser_ parses the incoming IPFIX packets.
 
-_Parsers_ and _Listeners_ must work together in a tight manner.
-Therefore they can not be combined in any possible combination.
-Which _Parser_ is supported by which _Listener_ is described in the documentation of each _Parser_ accordingly.
+_Parsers_ and _Listeners_ must work together closely.
+Therefore, they can not be combined in any random way.
+Which _Parser_ is supported by which _Listener_ is described in the documentation of each respective _Parser_.
 
-TIP: You can provide the parameter `maxClockSkew` in your parser definition to enable clock skew detection for exporters. It specifies the maximum delta in seconds between exporter and Minion timestamps. If violated an alarm will be generated for the exporting device. The default value is 0, so clock skew detection is disabled. Furthermore, a parameter `clockSkewEventRate` can be used to rate-limit clock skew events. The default is `3600` seconds, so every hour an event will be sent.
+TIP: You can provide the parameter `maxClockSkew` in your parser definition to enable clock skew detection for exporters. 
```

Comment:
While being here already, it is quite a *strange* place to mentioned it here. This feature is only supported by the netflow and ipfix parsers.

Reply:
I will remove it from here.